editor/tests/org.fusesource.ide.camel.editor.tests.integration/src/main/java/org/fusesource/ide/camel/editor/integration/globalconfiguration/dataformat/wizards/NewDataFormatWizardIT.java

```diff
@@ -101,7 +100,7 @@ public void testCreation() throws CoreException, IOException {
 	 * @param camelFile
 	 */
 	private void check(final String id, CamelFile camelFile) {
-		Assertions.assertThat(camelFile.getCamelContext().getDataformats().keySet()).containsExactly(id);
+		Assertions.assertThat(camelFile.getCamelContext().getDataformats().keySet()).contains(id);

```

Comment:
Can't remember why I changed that one. Seems to be not needed. It also works with containsExactly. Will revert that one.